packages/react-svg-core/src/index.ts

```diff
@@ -1,9 +1,9 @@
-import Svgo from "svgo";
 import { transformSync as babelTransform } from "@babel/core";
 import plugin from "babel-plugin-react-svg";
-
+import Svgo from "svgo/lib/svgo/jsAPI";
```

Comment:
It's not right. `svgo` package provides `optimize` named export which no longer output promise. Check this https://github.com/svg/svgo/releases/tag/v2.0.0